src/QMCWaveFunctions/BsplineFactory/createComplexSingle.cpp

```diff
@@ -70,8 +67,7 @@ std::unique_ptr<BsplineReaderBase> createBsplineComplexSingle(EinsplineSetBuilde
   }
 #else //QMC_COMPLEX
   app_summary() << "    Using real valued spline SPOs with complex single precision storage (C2R)." << std::endl;
-#if defined(ENABLE_OFFLOAD)
-  if (useGPU == "yes")
+  if (OMPTargetSelector::convertToPlatform(useGPU) == PlatformKind::OMPTARGET)
```

Comment:
As mentioned above the name seems wrong.

Reply:
Fixed accordingly